src/index.html

```diff
@@ -4,10 +4,553 @@
+            </div>
+            <div class="person__commasUp">
+              <img
+                src="/images/commas_blue.svg"
+                alt="commas for design"
+                class="person__commas"
+              >
+            </div>
+            <div class="person__feedback">AIR’s ideas are refreshing and out of the box. Authentic team that focuses on the important path of the brand.</div>
+            <div class="person__name">Tal Gilad</div>
+            <div class="person__occupation">Teach for America</div>
+          </div>
+          <div class="testimonials__person person">
+            <div class="person__image-wrap">
+              <div class="person__img-design person-yellow person--design-top"></div>
+              <div class="person__image">
+                <img
+                src="/images/testimonials/testimonials2.svg"
+                alt="Photo by Azadeh Hawkins"
+                class="person__photo">
+              </div>
+              <div class="person__img-design person-yellow person--design-bottom"></div>
+            </div>
+            <div class="person__commasUp">
+              <img
+                src="/images/commas/commas_yellow.svg"
+                alt="commas for design"
+                class="person__commas"
+              >
+            </div>
+            <div class="person__feedback">AIR is an exceptional agency that leads with creative talent, first-class account servicing.</div>
+            <div class="person__name">Azadeh Hawkins</div>
+            <div class="person__occupation">Hawkins Consulting</div>
+          </div>
+          <div class="testimonials__person person">
+            <div class="person__image-wrap">
+              <div class="person__img-design person-pink person--design-top"></div>
+              <div class="person__image">
+                <img
+                  src="/images/testimonials/testimonials3.svg"
+                  alt="Photo by Michel Grover"
+                  class="person__photo">
+              </div>
+              <div class="person__img-design person-pink person--design-bottom"></div>
+            </div>
+            <div class="person__commasUp">
+              <img
+                src="/images/commas/commas_pink.svg"
+                alt="commas for design"
+                class="person__commas">
+            </div>
+            <div class="person__feedback">AIR raises the agency bar to stratospheric heights on both creative output and client service.</div>
+            <div class="person__name">Michel Grover</div>
+            <div class="person__occupation">Hulu</div>
+          </div>
+        </div>
+      </section>
+
+      <section class="main__apply apply">
+        <div class="apply__block-wr">
+          <div class="apply__decor-element apply__decor-element_number1 decor-element decor-element_border-light decor-element_m decor-element_pink"></div>
```

Comment:
вроде тут все окей с БЭМ
```suggestion
          <div class="
            apply__decor-element
            apply__decor-element_number1
            decor-element
            decor-element_border-light
            decor-element_m 
            decor-element_pink
          "></div>
```

Reply:
Я пыталась так написать, но у меня выбило ошибку "error Duplicate class: class-no-dup"